src/pages/Chat/Chat.styled.jsx

```diff
@@ -22,6 +22,7 @@ export const HeaderChat = styled.div`
 
 export const HeaderChatImgContainer = styled.div`
   text-align: right;
+  margin-left: -80px;
```

Comment:
Please do not use negative margin-left with so big numbers, just use it for little adjustment of fewer than ten pixels. Please check here how the image is positioned correctly without using margin-left: 80px https://github.com/Azordev/maps-project/pull/72/commits/9304213eb9507aaf997f3441eab11a4b94ac0fb8